JustSaying/DefaultNamingStrategy.cs

```diff
@@ -5,19 +5,22 @@
 namespace JustSaying
 {
     /// <summary>
-    /// A default namign strategy for JustSaying bus.
+    /// A default naming strategy for JustSaying bus.
     /// Topic names are defaulted to message type name, lowercase (one topic per message type).
     /// Queue name is default to queue name.
     /// 
     /// Such configuration gives a queue for each IntoQueue configuration , and a queue is subcribed to multiple topics, where one topic per message.
     /// </summary>
     class DefaultNamingStrategy : INamingStrategy
     {
-        public string GetTopicName(string topicName, Type messageType) => messageType.ToTopicName().ToLower();
+        public string GetTopicName(string baseTopicName, Type messageType)
+            => string.IsNullOrEmpty(baseTopicName)
+                ? messageType.ToTopicName().ToLower()
+                : baseTopicName;
```

Comment:
Shouldn't this be concatenating the base topic name with the message type?

Reply:
I've just mirrored what we're doing in `GetQueueName(...)` below when a `BaseQueueName` is provided. If we change this, we should change both.

Back to Stu's original point, the whole base stuff is a bit weird (as well as the configuration syntax) so it could be a good thing to revisit when we plan JustSaying 7

Reply:
I see. However, from looking at this change alone (not digging any deeper) they both look wrong.

Reply:
Yea, it's a bit weird as base implies part of something not the actual "thing"